tests/js/tests/model/model-query.test.ts

```
@@ -0,0 +1,1284 @@
+  describe("include: edge cases — non-conforming linked nodes", () => {
+    @Model({ name: "EdgeComment" })
+    class EdgeComment extends Ad4mModel {
+      @Flag({ through: "ad4m://type", value: "ad4m://edge-comment" })
+      type!: string;
+
+      @Property({ through: "comment://text" })
+      text: string = "";
+    }
+
+    @Model({ name: "EdgeArticle" })
+    class EdgeArticle extends Ad4mModel {
```

Biome (2.4.4)

[error] This class is unsafely merged with an interface.

(lint/suspicious/noUnsafeDeclarationMerging)